src/OpenLoco/EditorController.cpp

```diff
@@ -109,13 +105,15 @@ namespace OpenLoco::EditorController
             return true;
         }
 
-        for (auto& town : TownManager::towns())
+        auto total = 0;
+        std::count(std::begin(TownManager::towns()), std::end(TownManager::towns()), total);
+
+        if (total != 0)
         {
-            if (!town.empty())
-                return true;
+            return true;
         }
```

Comment:
I think the original implementation (explicit for loop) is more readable, and more optimal. Best case, you'd only be looking at one town and returning, instead of checking all 128.

Reply:
It may be more optimal but it's not more readable. In the future i would just use .empty as these would be standard containers